cmd/otelcol/main.go

```diff
@@ -80,14 +85,21 @@ func main() {
 		},
 	}
 
-	os.Args = append(os.Args[:1], collectorSettings.ColCoreArgs()...)
+	allArgs := args[:1]
+	allArgs = append(allArgs, collectorSettings.ColCoreArgs()...)
+	os.Args = allArgs
```

Comment:
why this change? can't see a difference

Reply:
Lint doesn't like `append` that doesn't assign resulting slice to the first parameter so logically what I want is `os.Args = append(args[:1], collectorSettings.ColCoreArgs()...)`, but, lint is not happy that `args` is not on the left-hand side.